src/Middleware/AuthorizationMiddleware.php

```diff
@@ -143,43 +116,22 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         return $response;
     }
 
-    /**
-     * Returns unauthorized handler.
-     *
-     * @return \Authorization\Middleware\UnauthorizedHandler\HandlerInterface
-     */
-    protected function getHandler(): HandlerInterface
-    {
-        $handler = $this->getConfig('unauthorizedHandler');
-        if (!is_array($handler)) {
-            $handler = [
-                'className' => $handler,
-            ];
-        }
-        if (!isset($handler['className'])) {
-            throw new RuntimeException('Missing `className` key from handler config.');
-        }
-
-        return HandlerFactory::create($handler['className']);
-    }
-
     /**
      * Returns AuthorizationServiceInterface instance.
      *
      * @param \Psr\Http\Message\ServerRequestInterface $request Server request.
      * @return \Authorization\AuthorizationServiceInterface
      * @throws \RuntimeException When authorization method has not been defined.
      */
-    protected function getAuthorizationService(
-        ServerRequestInterface $request
-    ): AuthorizationServiceInterface {
+    protected function getAuthorizationService(ServerRequestInterface $request): AuthorizationServiceInterface
+    {
         $service = $this->subject;
         if ($this->subject instanceof AuthorizationServiceProviderInterface) {
             $service = $this->subject->getAuthorizationService($request);
         }
 
         if (!$service instanceof AuthorizationServiceInterface) {
-            throw new RuntimeException(sprintf(
+            throw new \RuntimeException(sprintf(
```

Comment:
We prefer to have `use RuntimeException` at the top of the file instead of inline qualifiers.

Reply:
Done